tasks/plugins.yml

```diff
@@ -26,7 +26,8 @@
 
 - name: Install Jenkins plugins using password.
   jenkins_plugin:
-    name: "{{ item }}"
+    name: "{{ item.name | default(item) }}"
```

Comment:
And one last nitpick—since we're making `jenkins_plugins` a little more flexible, can you supplement the documentation for this variable in the README so it accounts for either just listing the role names (legacy), or adding 'name' and 'version' keys? If not, I can do it after a merge.